content/en/observability_pipelines/guide/remap_reserved_attributes.md

```diff
@@ -71,8 +73,96 @@ In the below example image, the input shows `status` with the value `wrongstatus
 
 {{< img src="observability_pipelines/guide/remap_attributes/custom_processor_statically_assign.png" alt="A custom processor showing an input the incorrect status value and the output showing the correct status" style="width:100%;" >}}
 
-## Further reading
+## Remap source and service attributes when using the Splunk HEC source and Datadog destination
+
+Follow the instructions in this section to remap the `source` and/or `service` values if you are using a Splunk HEC source and Datadog destination. You must follow these instructions to remap those attributes because:
+
+ - Splunk's `service` is what Datadog calls the `source` attribute.
+ - Splunk's `sourcetype` is what Datadog calls the `ddsource` attribute.
+
+**Note**: If you want to remap other reserved attributes, such as `env` and `hostname`, follow the [Remap the value of reserved attributes](#remap-the-value-of-reserved-attributes) instructions.
+
+You can use the [Custom Processor](#remap-service-and-source-attributes-using-the-custom-processor) or [Edit Fields](#remap-service-and-source-attributes-using-edit-fields) to:
+
+1. Remap the input log's `service` field to the `source` field name.
+1. Remap the input log's `source` field to the `ddsource` field name.
+
+### Remap service and source attributes using the Custom Processor
+
+This is an example input log from the Splunk HEC source:
+
+```json
+{
+  "service": "wrongService"
+  "source": "wrongSource"
+}
+```
+
+If these are the correct values you want for the log sent to Datadog:
+
+```json
+{
+  "ddsource": "akamai",
+  "source": "cdn-logs"
+}
+```
```

Comment:
Nit: I think this could be easier to follow as a simple statement rather than a conditional "If" phrase, which feels like an incomplete fragment and requires the reader to follow its logic across several lines. Maybe something more like:
```suggestion
Assume these are the correct values you want for the log sent to Datadog:

```json
{
  "ddsource": "akamai",
  "source": "cdn-logs"
}
```
```